scripts/build-probe-binaries

```diff
@@ -80,6 +82,61 @@ function build_sysdig {
 	cd $BASEDIR
 }
 
+function build_falco {
+
+	if [ ! -f $BASEDIR/output/$PROBE_NAME-$PROBE_VERSION-$ARCH-$KERNEL_RELEASE-$HASH.ko ] || [ ! -f $BASEDIR/output/$PROBE_NAME-$PROBE_VERSION-$ARCH-$KERNEL_RELEASE-$HASH_ORIG.ko ]; then
+
+		echo Building $PROBE_NAME-$PROBE_VERSION-$ARCH-$KERNEL_RELEASE-$HASH.ko [${FUNCNAME[1]}]
+
+		if [ ! -d sysdig ]; then
```

Comment:
The work of cloning, pulling, and checking out the correct tag is mostly common to all 3 builds. Can we put that in a separate function? The build module process is both fragile and a big pain point, and the code duplication is getting excessive.